src/notebook/widget.ts

```diff
@@ -154,6 +154,22 @@ const TOOLBAR_PRESSED = 'jp-mod-pressed';
 const TOOLBAR_BUSY = 'jp-mod-busy';
 
 /**
+ * The class name added to a notebook in edit mode.
+ */
+const EDIT_CLASS = 'jp-mod-editMode';
+
+/**
+ * The class name added to a notebook in command mode.
+ */
+const COMMAND_CLASS = 'jp-mod-commandMode';
+
+/**
+ * The class name added to Editor widget instances.
+ * Keep in sync with `editor/widget.ts`.
+ */
+const EDITOR_CLASS = 'jp-Editor';
```

Comment:
Perhaps we should export this from the editor file? On the other hand, we wouldn't be able to import the const into a CSS file, so we already have to keep things in sync with the CSS files by hand.